src/test/kotlin/com/vanniktech/rxriddles/Riddle35Test.kt

```diff
@@ -0,0 +1,35 @@
+package com.vanniktech.rxriddles
+
+import com.vanniktech.rxriddles.solutions.Riddle35Solution
+import com.vanniktech.rxriddles.tools.RxRule
+import io.reactivex.Maybe
+import io.reactivex.Single
+import org.junit.Rule
+import org.junit.Test
+
+/** Solution [Riddle35Solution] */
+class Riddle35Test {
+
+  @get:Rule
```

Comment:
can you have the same formatting as the other riddles?

Reply:
Is there a particular code-style XML we can import into match yours? Or perhaps include the codestyle in the repository if it's custom?

Reply:
It's the normal square codestyle.